routing/control_tower_test.go

```diff
@@ -300,7 +300,9 @@ func initDB(keepFailedPaymentAttempts bool) (*channeldb.DB, error) {
 		return nil, err
 	}
 
-	db, err := channeldb.Open(tempPath, channeldb.OptionKeepFailedPaymentAttempts(keepFailedPaymentAttempts))
+	db, err := channeldb.Open(tempPath,
```

Comment:
This should be fixed in the previous commit.
And the correct formatting would be:
```
	db, err := channeldb.Open(
		tempPath, channeldb.OptionKeepFailedPaymentAttempts(
			keepFailedPaymentAttempts,
		).
	)
```

Reply:
Done.